### ISSUE TITLE
Add support for hdfs file system shell commands

### ISSUE THREAD

**Issue by santosh-d3vpl3x** [Kimahriman/hdfs-native#171]
It would be great to support shell commands from https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/FileSystemShell.html

I believe this implementation should be straightforward for python.

Found an example where this is done for golang  https://github.com/colinmarc/hdfs

**Comment by Kimahriman** [comment]
Yeah I've thought about this, should be fairly straightforward to do in Python. One thing that would need to be decided is whether to try to make it similar to the Hadoop CLI, or make it more based on the standard Linux CLI. Also, the main thing missing from this library currently is any kind of glob handling that a lot of the CLI supports.

**Comment by santosh-d3vpl3x** [comment]
> One thing that would need to be decided is whether to try to make it similar to the Hadoop CLI, or make it more based on the standard Linux CLI. 

It is probably better to keep these two as close to each other as possible unless the cost of keeping them close is too high. I see this CLI more as a replacement of Hadoop CLI in containers, especially in jupyter environment. That being said, I am not opposed to the idea of alternatives either :)

**Comment by Kimahriman** [comment]
Created a milestone for this https://github.com/Kimahriman/hdfs-native/milestone/2 and will make individual issues to work of each command.

**Comment by Kimahriman** [comment]
Cut an initial release with some CLI behavior, unfortunately without globbing support yet as that will take some work to convert over the Hadoop implementation
